conf/estates.routes

```diff
@@ -5,3 +5,5 @@ POST        /check                    uk.gov.hmrc.trusts.controllers.CheckEstate
 POST        /register                 uk.gov.hmrc.trusts.controllers.RegisterEstateController.registration
 
 GET         /:utr                     uk.gov.hmrc.trusts.controllers.GetEstateController.get(utr: String)
+
+POST        /estatevariation          uk.gov.hmrc.trusts.controllers.EstateVariationsController.estateVariation
```

Comment:
The endpoint should just be `/variation`

Reply:
Changed to variation